settings.gradle

```diff
@@ -52,6 +52,23 @@ if(!System.getenv().containsKey("SUB_BUILD") || System.getenv().get("SUB_BUILD")
     include ':airbyte-server'
     include ':airbyte-webapp'
     include ':airbyte-tests'
+
+    // acceptance tests
```

Comment:
@cgardens I had to pull all these in in the platform build so acceptance tests can run. Lmk if there is a better way of doing this.

Reply:
@davinchia can you help me understand why we need these? my understanding is that the acceptance tests us pinned versions of postgres and normalization (i.e. version docker images that have already been pushed to docker hub). I'm missing why there is a build dependency here. What was the error you were running into?

Reply:
ah. i see. i didn't see this because the acceptance tests don't run as part of the PR build.

Reply:
i will debug it.

Reply:
I just tried to reproduce this to make sure I wasn't going crazy.

I removed the dependencies under `// acceptance tests` , ran `./tools/bin/acceptance_test.sh` and got this error:
```
➜  airbyte git:(master) ✗ ./tools/bin/acceptance_test.sh
Starting app...
Creating network "airbyte_default" with the default driver
Creating volume "airbyte_workspace" with default driver
Creating volume "airbyte_data" with default driver
Creating volume "airbyte_db" with default driver
Creating airbyte-db        ... done
Creating init              ... done
Creating airbyte-webapp    ... done
Creating airbyte-server    ... done
Creating airbyte-data-seed ... done
Creating airbyte-temporal  ... done
Creating airbyte-scheduler ... done
Waiting for services to begin
Running e2e tests via gradle
Building Airbyte Sub Build: PLATFORM

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':airbyte-tests:acceptanceTests'.
> Task with path ':airbyte-integrations:connectors:source-e2e-test:airbyteDocker' not found in project ':airbyte-tests'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.7.1/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 3s

Publishing build scan...
https://gradle.com/s/n53awh5lwvpaq

docker-compose logs:
Attaching to init, airbyte-server, airbyte-temporal, airbyte-db, airbyte-scheduler, airbyte-data-seed, airbyte-webapp
airbyte-temporal    | 2021-07-23T04:29:40.517424700Z {"level":"info","ts":"2021-07-23T04:29:40.517Z","msg":"Current reachable members","service":"frontend","component":"service-resolver","service":"matching","addresses":"[172.19.0.6:7235]","logging-call-at":"rpServiceResolver.go:266"}
airbyte-temporal    | 2021-07-23T04:29:40.517759800Z {"level":"info","ts":"2021-07-23T04:29:40.517Z","msg":"Received a ring changed event","service":"history","component":"service-resolver","service":"matching","logging-call-at":"rpServiceResolver.go:219"}
airbyte-temporal    | 2021-07-23T04:29:40.517787700Z {"level":"info","ts":"2021-07-23T04:29:40.517Z","msg":"Received a ring changed event","service":"history","component":"service-resolver","service":"history","logging-call-at":"rpServiceResolver.go:219"}
airbyte-temporal    | 2021-07-23T04:29:40.517907800Z {"level":"info","ts":"2021-07-23T04:29:40.517Z","msg":"none","service":"history","component":"shard-controller","address":"172.19.0.6:7234","shard-update":"RingMembershipChangedEvent","number-processed":1,"number-deleted":0,"number":0,"logging-call-at":"controller_impl.go:326"}
airbyte-temporal    | 2021-07-23T04:29:40.517924500Z {"level":"info","ts":"2021-07-23T04:29:40.517Z","msg":"Received a ring changed event","service":"history","component":"service-resolver","service":"worker","logging-call-at":"rpServiceResolver.go:219"}
airbyte-temporal    | 2021-07-23T04:29:40.518070700Z {"level":"info","ts":"2021-07-23T04:29:40.517Z","msg":"Received a ring changed event","service":"history","component":"service-resolver","service":"frontend","logging-call-at":"rpServiceResolver.go:219"}
airbyte-temporal    | 2021-07-23T04:29:40.518195300Z {"level":"info","ts":"2021-07-23T04:29:40.517Z","msg":"Current reachable members","service":"frontend","component":"service-resolver","service":"history","addresses":"[172.19.0.6:7234]","logging-call-at":"rpServiceResolver.go:266"}
airbyte-temporal    | 2021-07-23T04:29:40.519895900Z {"level":"info","ts":"2021-07-23T04:29:40.519Z","msg":"Membership heartbeat upserted successfully","service":"worker","address":"172.19.0.6","port":6939,"hostId":"98a9c045-eb6e-11eb-a2b5-0242ac130006","logging-call-at":"rpMonitor.go:222"}
airbyte-temporal    | 2021-07-23T04:29:40.522799300Z {"level":"info","ts":"2021-07-23T04:29:40.522Z","msg":"Service resources started","service":"frontend","address":"172.19.0.6:7233","logging-call-at":"resourceImpl.go:381"}
airbyte-temporal    | 2021-07-23T04:29:40.523169900Z {"level":"info","ts":"2021-07-23T04:29:40.522Z","msg":"Get dynamic config","name":"frontend.enableServerVersionCheck","value":"true","default-value":"true","logging-call-at":"config.go:79"}
airbyte-temporal    | 2021-07-23T04:29:40.523561500Z {"level":"info","ts":"2021-07-23T04:29:40.523Z","msg":"Starting to serve on frontend listener","service":"frontend","logging-call-at":"service.go:276"}
airbyte-temporal    | 2021-07-23T04:29:40.523581700Z {"level":"info","ts":"2021-07-23T04:29:40.523Z","msg":"bootstrap hosts fetched","service":"worker","bootstrap-hostports":"172.19.0.6:6934,172.19.0.6:6935,172.19.0.6:6933,172.19.0.6:6939","logging-call-at":"rpMonitor.go:263"}
airbyte-temporal    | 2021-07-23T04:29:40.529957800Z {"level":"info","ts":"2021-07-23T04:29:40.529Z","msg":"Current reachable members","service":"worker","component":"service-resolver","service":"history","addresses":"[172.19.0.6:7234]","logging-call-at":"rpServiceResolver.go:266"}
airbyte-temporal    | 2021-07-23T04:29:40.530304700Z {"level":"info","ts":"2021-07-23T04:29:40.530Z","msg":"Current reachable members","service":"worker","component":"service-resolver","service":"worker","addresses":"[172.19.0.6:7239]","logging-call-at":"rpServiceResolver.go:266"}
airbyte-temporal    | 2021-07-23T04:29:40.530700800Z {"level":"info","ts":"2021-07-23T04:29:40.530Z","msg":"Current reachable members","service":"worker","component":"service-resolver","service":"frontend","addresses":"[172.19.0.6:7233]","logging-call-at":"rpServiceResolver.go:266"}
airbyte-temporal    | 2021-07-23T04:29:40.531046000Z {"level":"info","ts":"2021-07-23T04:29:40.530Z","msg":"Received a ring changed event","service":"history","component":"service-resolver","service":"worker","logging-call-at":"rpServiceResolver.go:219"}
airbyte-temporal    | 2021-07-23T04:29:40.531068000Z {"level":"info","ts":"2021-07-23T04:29:40.530Z","msg":"Received a ring changed event","service":"history","component":"service-resolver","service":"frontend","logging-call-at":"rpServiceResolver.go:219"}
airbyte-temporal    | 2021-07-23T04:29:40.531087400Z {"level":"info","ts":"2021-07-23T04:29:40.530Z","msg":"Received a ring changed event","service":"history","component":"service-resolver","service":"matching","logging-call-at":"rpServiceResolver.go:219"}
airbyte-temporal    | 2021-07-23T04:29:40.531235400Z {"level":"info","ts":"2021-07-23T04:29:40.531Z","msg":"Received a ring changed event","service":"history","component":"service-resolver","service":"history","logging-call-at":"rpServiceResolver.go:219"}
airbyte-temporal    | 2021-07-23T04:29:40.531292700Z {"level":"info","ts":"2021-07-23T04:29:40.531Z","msg":"none","service":"history","component":"shard-controller","address":"172.19.0.6:7234","shard-update":"RingMembershipChangedEvent","number-processed":1,"number-deleted":0,"number":0,"logging-call-at":"controller_impl.go:326"}
airbyte-temporal    | 2021-07-23T04:29:40.531308700Z {"level":"info","ts":"2021-07-23T04:29:40.531Z","msg":"Current reachable members","service":"history","component":"service-resolver","service":"worker","addresses":"[172.19.0.6:7239]","logging-call-at":"rpServiceResolver.go:266"}
airbyte-temporal    | 2021-07-23T04:29:40.531572700Z {"level":"info","ts":"2021-07-23T04:29:40.531Z","msg":"Current reachable members","service":"worker","component":"service-resolver","service":"matching","addresses":"[172.19.0.6:7235]","logging-call-at":"rpServiceResolver.go:266"}
airbyte-temporal    | 2021-07-23T04:29:40.535905800Z {"level":"info","ts":"2021-07-23T04:29:40.535Z","msg":"Service resources started","service":"worker","address":"172.19.0.6:7239","logging-call-at":"resourceImpl.go:381"}
airbyte-temporal    | 2021-07-23T04:29:40.539107800Z {"level":"info","ts":"2021-07-23T04:29:40.538Z","msg":"Get dynamic config","name":"worker.executionsScannerEnabled","value":"false","default-value":"false","logging-call-at":"config.go:79"}
airbyte-temporal    | 2021-07-23T04:29:40.539132100Z {"level":"info","ts":"2021-07-23T04:29:40.539Z","msg":"Get dynamic config","name":"worker.taskQueueScannerEnabled","value":"true","default-value":"true","logging-call-at":"config.go:79"}
airbyte-temporal    | 2021-07-23T04:29:40.551476300Z {"level":"info","ts":"2021-07-23T04:29:40.551Z","msg":"Received a ring changed event","service":"matching","component":"service-resolver","service":"frontend","logging-call-at":"rpServiceResolver.go:219"}
airbyte-temporal    | 2021-07-23T04:29:40.551527900Z {"level":"info","ts":"2021-07-23T04:29:40.551Z","msg":"Received a ring changed event","service":"matching","component":"service-resolver","service":"matching","logging-call-at":"rpServiceResolver.go:219"}
airbyte-temporal    | 2021-07-23T04:29:40.551544600Z {"level":"info","ts":"2021-07-23T04:29:40.551Z","msg":"Received a ring changed event","service":"matching","component":"service-resolver","service":"history","logging-call-at":"rpServiceResolver.go:219"}
airbyte-scheduler   | 2021-07-23T04:29:37.292308900Z 2021-07-23 04:29:37 INFO i.a.s.a.SchedulerApp(main):199 - {workspace_app_root=/tmp/workspace/scheduler/logs} - workspaceRoot = /tmp/workspace
airbyte-scheduler   | 2021-07-23T04:29:37.355065700Z 2021-07-23 04:29:37 INFO i.a.s.a.SchedulerApp(main):202 - {workspace_app_root=/tmp/workspace/scheduler/logs} - temporalHost = airbyte-temporal:7233
airbyte-scheduler   | 2021-07-23T04:29:37.356269300Z 2021-07-23 04:29:37 INFO i.a.s.a.SchedulerApp(main):204 - {workspace_app_root=/tmp/workspace/scheduler/logs} - Creating Job DB connection pool...
airbyte-scheduler   | 2021-07-23T04:29:37.368864300Z 2021-07-23 04:29:37 WARN i.a.d.Databases(createPostgresDatabaseWithRetry):85 - {workspace_app_root=/tmp/workspace/scheduler/logs} - Waiting for database to become available...
airbyte-scheduler   | 2021-07-23T04:29:37.948072600Z 2021-07-23 04:29:37 INFO i.a.d.Databases(createPostgresDatabaseWithRetry):90 - {workspace_app_root=/tmp/workspace/scheduler/logs} - Database is not ready yet. Please wait a moment, it might still be initializing...
airbyte-scheduler   | 2021-07-23T04:29:42.953860000Z 2021-07-23 04:29:42 WARN i.a.d.Databases(createPostgresDatabaseWithRetry):85 - {workspace_app_root=/tmp/workspace/scheduler/logs} - Waiting for database to become available...
airbyte-scheduler   | 2021-07-23T04:29:43.155777200Z 2021-07-23 04:29:43 INFO i.a.d.Databases(createPostgresDatabaseWithRetry):102 - {workspace_app_root=/tmp/workspace/scheduler/logs} - Database available!
airbyte-scheduler   | 2021-07-23T04:29:43.159059500Z 2021-07-23 04:29:43 INFO i.a.c.EnvConfigs(getEnvOrDefault):302 - {workspace_app_root=/tmp/workspace/scheduler/logs} - WORKER_ENVIRONMENT not found or empty, defaulting to DOCKER
airbyte-scheduler   | 2021-07-23T04:29:43.164161000Z 2021-07-23 04:29:43 INFO i.a.c.EnvConfigs(getEnvOrDefault):302 - {workspace_app_root=/tmp/workspace/scheduler/logs} - DOCKER_NETWORK not found or empty, defaulting to host
airbyte-scheduler   | 2021-07-23T04:29:43.188279800Z 2021-07-23 04:29:43 INFO i.a.c.p.ConfigPersistenceBuilder(getDbPersistenceWithFileSeed):96 - {workspace_app_root=/tmp/workspace/scheduler/logs} - Creating db-based config persistence, and loading seed and existing data from files
airbyte-scheduler   | 2021-07-23T04:29:43.194840400Z 2021-07-23 04:29:43 INFO i.a.c.p.FileSystemConfigPersistence(createWithValidation):62 - {workspace_app_root=/tmp/workspace/scheduler/logs} - Constructing file system config persistence (root: /data)
airbyte-scheduler   | 2021-07-23T04:29:43.195856300Z 2021-07-23 04:29:43 INFO i.a.c.p.FileSystemConfigPersistence(createWithValidation):71 - {workspace_app_root=/tmp/workspace/scheduler/logs} - Config volume is ready (waiting time: 0 s)
airbyte-scheduler   | 2021-07-23T04:29:43.396967100Z 2021-07-23 04:29:43 INFO i.a.c.p.ConfigPersistenceBuilder(getDbPersistence):106 - {workspace_app_root=/tmp/workspace/scheduler/logs} - Use database config persistence.
airbyte-scheduler   | 2021-07-23T04:29:43.397813700Z 2021-07-23 04:29:43 INFO i.a.c.EnvConfigs(getEnvOrDefault):302 - {workspace_app_root=/tmp/workspace/scheduler/logs} - CONFIG_DATABASE_USER not found or empty, defaulting to docker
airbyte-scheduler   | 2021-07-23T04:29:43.398762000Z 2021-07-23 04:29:43 INFO i.a.c.EnvConfigs(getEnvOrDefault):302 - {workspace_app_root=/tmp/workspace/scheduler/logs} - CONFIG_DATABASE_PASSWORD not found or empty, defaulting to docker
airbyte-scheduler   | 2021-07-23T04:29:43.399638300Z 2021-07-23 04:29:43 INFO i.a.c.EnvConfigs(getEnvOrDefault):302 - {workspace_app_root=/tmp/workspace/scheduler/logs} - CONFIG_DATABASE_URL not found or empty, defaulting to jdbc:postgresql://db:5432/airbyte
airbyte-scheduler   | 2021-07-23T04:29:43.400638900Z 2021-07-23 04:29:43 WARN i.a.d.Databases(createPostgresDatabaseWithRetry):85 - {workspace_app_root=/tmp/workspace/scheduler/logs} - Waiting for database to become available...
airbyte-scheduler   | 2021-07-23T04:29:43.401595900Z 2021-07-23 04:29:43 INFO i.a.d.Databases(lambda$static$4):67 - {workspace_app_root=/tmp/workspace/scheduler/logs} - Testing if airbyte_configs has been created...
airbyte-scheduler   | 2021-07-23T04:29:43.430158900Z 2021-07-23 04:29:43 INFO i.a.d.Databases(createPostgresDatabaseWithRetry):90 - {workspace_app_root=/tmp/workspace/scheduler/logs} - Database is not ready yet. Please wait a moment, it might still be initializing...
airbyte-scheduler   | 2021-07-23T04:29:48.431963100Z 2021-07-23 04:29:48 WARN i.a.d.Databases(createPostgresDatabaseWithRetry):85 - {workspace_app_root=/tmp/workspace/scheduler/logs} - Waiting for database to become available...
airbyte-scheduler   | 2021-07-23T04:29:48.432786200Z 2021-07-23 04:29:48 INFO i.a.d.Databases(lambda$static$4):67 - {workspace_app_root=/tmp/workspace/scheduler/logs} - Testing if airbyte_configs has been created...
airbyte-scheduler   | 2021-07-23T04:29:48.448374800Z 2021-07-23 04:29:48 INFO i.a.d.Databases(createPostgresDatabaseWithRetry):102 - {workspace_app_root=/tmp/workspace/scheduler/logs} - Database available!
airbyte-scheduler   | 2021-07-23T04:29:48.453228900Z 2021-07-23 04:29:48 INFO i.a.c.EnvConfigs(getEnvOrDefault):302 - {workspace_app_root=/tmp/workspace/scheduler/logs} - MINIMUM_WORKSPACE_RETENTION_DAYS not found or empty, defaulting to 1
airbyte-scheduler   | 2021-07-23T04:29:48.454165800Z 2021-07-23 04:29:48 INFO i.a.c.EnvConfigs(getEnvOrDefault):302 - {workspace_app_root=/tmp/workspace/scheduler/logs} - MAXIMUM_WORKSPACE_RETENTION_DAYS not found or empty, defaulting to 60
airbyte-scheduler   | 2021-07-23T04:29:48.454890600Z 2021-07-23 04:29:48 INFO i.a.c.EnvConfigs(getEnvOrDefault):302 - {workspace_app_root=/tmp/workspace/scheduler/logs} - MAXIMUM_WORKSPACE_SIZE_MB not found or empty, defaulting to 5000
airbyte-scheduler   | 2021-07-23T04:29:48.459499900Z 2021-07-23 04:29:48 INFO i.a.c.EnvConfigs(getEnvOrDefault):302 - {workspace_app_root=/tmp/workspace/scheduler/logs} - WORKER_ENVIRONMENT not found or empty, defaulting to DOCKER
airbyte-scheduler   | 2021-07-23T04:29:48.461141400Z 2021-07-23 04:29:48 INFO i.a.c.EnvConfigs(getEnvOrDefault):302 - {workspace_app_root=/tmp/workspace/scheduler/logs} - WORKER_ENVIRONMENT not found or empty, defaulting to DOCKER
airbyte-scheduler   | 2021-07-23T04:29:48.856050600Z 2021-07-23 04:29:48 INFO i.t.s.WorkflowServiceStubsImpl(<init>):168 - {workspace_app_root=/tmp/workspace/scheduler/logs} - Created GRPC client for channel: ManagedChannelOrphanWrapper{delegate=ManagedChannelImpl{logId=1, target=airbyte-temporal:7233}}
airbyte-scheduler   | 2021-07-23T04:29:48.856981000Z 2021-07-23 04:29:48 INFO i.a.w.t.TemporalUtils(waitForTemporalServerAndLog):132 - {workspace_app_root=/tmp/workspace/scheduler/logs} - Waiting for temporal server...
airbyte-scheduler   | 2021-07-23T04:29:48.857673100Z 2021-07-23 04:29:48 WARN i.a.w.t.TemporalUtils(waitForTemporalServerAndLog):137 - {workspace_app_root=/tmp/workspace/scheduler/logs} - Waiting for default namespace to be initialized in temporal...
airbyte-server      | 2021-07-23T04:29:42.173173300Z 2021-07-23 04:29:42 INFO i.a.s.ServerApp(runServer):212 - {workspace_app_root=/tmp/workspace/server/logs} - Creating config repository...
airbyte-server      | 2021-07-23T04:29:42.238330500Z 2021-07-23 04:29:42 INFO i.a.c.p.ConfigPersistenceBuilder(getDbPersistenceWithFileSeed):96 - {workspace_app_root=/tmp/workspace/server/logs} - Creating db-based config persistence, and loading seed and existing data from files
airbyte-server      | 2021-07-23T04:29:42.248717800Z 2021-07-23 04:29:42 INFO i.a.c.p.FileSystemConfigPersistence(createWithValidation):62 - {workspace_app_root=/tmp/workspace/server/logs} - Constructing file system config persistence (root: /data)
airbyte-server      | 2021-07-23T04:29:42.249560400Z 2021-07-23 04:29:42 INFO i.a.c.p.FileSystemConfigPersistence(createWithValidation):71 - {workspace_app_root=/tmp/workspace/server/logs} - Config volume is ready (waiting time: 0 s)
airbyte-server      | 2021-07-23T04:29:42.416824400Z 2021-07-23 04:29:42 INFO i.a.c.p.ConfigPersistenceBuilder(getDbPersistence):106 - {workspace_app_root=/tmp/workspace/server/logs} - Use database config persistence.
airbyte-server      | 2021-07-23T04:29:42.418308800Z 2021-07-23 04:29:42 INFO i.a.c.EnvConfigs(getEnvOrDefault):302 - {workspace_app_root=/tmp/workspace/server/logs} - CONFIG_DATABASE_USER not found or empty, defaulting to docker
airbyte-server      | 2021-07-23T04:29:42.419197900Z 2021-07-23 04:29:42 INFO i.a.c.EnvConfigs(getEnvOrDefault):302 - {workspace_app_root=/tmp/workspace/server/logs} - CONFIG_DATABASE_PASSWORD not found or empty, defaulting to docker
airbyte-server      | 2021-07-23T04:29:42.420097600Z 2021-07-23 04:29:42 INFO i.a.c.EnvConfigs(getEnvOrDefault):302 - {workspace_app_root=/tmp/workspace/server/logs} - CONFIG_DATABASE_URL not found or empty, defaulting to jdbc:postgresql://db:5432/airbyte
airbyte-server      | 2021-07-23T04:29:42.429447500Z 2021-07-23 04:29:42 WARN i.a.d.Databases(createPostgresDatabaseWithRetry):85 - {workspace_app_root=/tmp/workspace/server/logs} - Waiting for database to become available...
airbyte-server      | 2021-07-23T04:29:42.456743800Z 2021-07-23 04:29:42 INFO i.a.d.Databases(lambda$static$2):58 - {workspace_app_root=/tmp/workspace/server/logs} - Testing config database connection...
airbyte-server      | 2021-07-23T04:29:43.135919300Z 2021-07-23 04:29:43 INFO i.a.d.Databases(createPostgresDatabaseWithRetry):102 - {workspace_app_root=/tmp/workspace/server/logs} - Database available!
airbyte-server      | 2021-07-23T04:29:43.158094100Z 2021-07-23 04:29:43 INFO i.a.c.p.DatabaseConfigPersistence(lambda$initialize$0):81 - {workspace_app_root=/tmp/workspace/server/logs} - Config database has not been initialized
airbyte-server      | 2021-07-23T04:29:43.159102900Z 2021-07-23 04:29:43 INFO i.a.c.p.DatabaseConfigPersistence(lambda$initialize$0):82 - {workspace_app_root=/tmp/workspace/server/logs} - Creating tables with schema: -- tables
airbyte-server      | 2021-07-23T04:29:43.159120700Z  CREATE
airbyte-server      | 2021-07-23T04:29:43.159137500Z     TABLE
airbyte-server      | 2021-07-23T04:29:43.159152500Z         IF NOT EXISTS AIRBYTE_CONFIGS(
airbyte-server      | 2021-07-23T04:29:43.159167200Z             id BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
airbyte-server      | 2021-07-23T04:29:43.159181700Z             config_id VARCHAR(36) NOT NULL,
airbyte-server      | 2021-07-23T04:29:43.159194500Z             config_type VARCHAR(60) NOT NULL,
airbyte-server      | 2021-07-23T04:29:43.159210800Z             config_blob JSONB NOT NULL,
airbyte-server      | 2021-07-23T04:29:43.159285900Z             created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
airbyte-server      | 2021-07-23T04:29:43.159302200Z             updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP
airbyte-server      | 2021-07-23T04:29:43.159319500Z         );
airbyte-server      | 2021-07-23T04:29:43.159330800Z
airbyte-server      | 2021-07-23T04:29:43.159343300Z -- indices
airbyte-server      | 2021-07-23T04:29:43.159359600Z  CREATE
airbyte-server      | 2021-07-23T04:29:43.159375400Z     UNIQUE INDEX IF NOT EXISTS airbyte_configs_type_id_idx ON
airbyte-server      | 2021-07-23T04:29:43.159390200Z     AIRBYTE_CONFIGS(
airbyte-server      | 2021-07-23T04:29:43.159405700Z         config_type,
airbyte-server      | 2021-07-23T04:29:43.159419300Z         config_id
airbyte-server      | 2021-07-23T04:29:43.159434900Z     );
airbyte-server      | 2021-07-23T04:29:43.159447800Z
airbyte-server      | 2021-07-23T04:29:43.159463200Z CREATE
airbyte-server      | 2021-07-23T04:29:43.159478600Z     INDEX IF NOT EXISTS airbyte_configs_id_idx ON
airbyte-server      | 2021-07-23T04:29:43.159493900Z     AIRBYTE_CONFIGS(config_id);
airbyte-server      | 2021-07-23T04:29:43.159508800Z
airbyte-server      | 2021-07-23T04:29:43.191438000Z 2021-07-23 04:29:43 INFO i.a.c.p.DatabaseConfigPersistence(lambda$loadData$3):101 - {workspace_app_root=/tmp/workspace/server/logs} - Loading data to config database...
airbyte-server      | 2021-07-23T04:29:43.377107600Z 2021-07-23 04:29:43 INFO i.a.c.p.DatabaseConfigPersistence(insertConfigRecord):222 - {workspace_app_root=/tmp/workspace/server/logs} - Inserting STANDARD_WORKSPACE record 5ae6b09b-fdec-41af-aaf7-7d94cfc33ef6
airbyte-server      | 2021-07-23T04:29:43.407550200Z 2021-07-23 04:29:43 INFO i.a.c.p.DatabaseConfigPersistence(insertConfigRecord):222 - {workspace_app_root=/tmp/workspace/server/logs} - Inserting STANDARD_SOURCE_DEFINITION record 253487c0-2246-43ba-a21f-5116b20a2c50
airbyte-server      | 2021-07-23T04:29:43.410865400Z 2021-07-23 04:29:43 INFO i.a.c.p.DatabaseConfigPersistence(insertConfigRecord):222 - {workspace_app_root=/tmp/workspace/server/logs} - Inserting STANDARD_SOURCE_DEFINITION record 9da77001-af33-4bcd-be46-6252bf9342b9
airbyte-server      | 2021-07-23T04:29:43.413075000Z 2021-07-23 04:29:43 INFO i.a.c.p.DatabaseConfigPersistence(insertConfigRecord):222 - {workspace_app_root=/tmp/workspace/server/logs} - Inserting STANDARD_SOURCE_DEFINITION record 9fa5862c-da7c-11eb-8d19-0242ac130003
airbyte-server      | 2021-07-23T04:29:43.415200800Z 2021-07-23 04:29:43 INFO i.a.c.p.DatabaseConfigPersistence(insertConfigRecord):222 - {workspace_app_root=/tmp/workspace/server/logs} - Inserting STANDARD_SOURCE_DEFINITION record decd338e-5647-4c0b-adf4-da0e75f5a750
airbyte-server      | 2021-07-23T04:29:43.418209200Z 2021-07-23 04:29:43 INFO i.a.c.p.DatabaseConfigPersistence(insertConfigRecord):222 - {workspace_app_root=/tmp/workspace/server/logs} - Inserting STANDARD_SOURCE_DEFINITION record b5ea17b1-f170-46dc-bc31-cc744ca984c1
airbyte-server      | 2021-07-23T04:29:43.420886000Z 2021-07-23 04:29:43 INFO i.a.c.p.DatabaseConfigPersistence(insertConfigRecord):222 - {workspace_app_root=/tmp/workspace/server/logs} - Inserting STANDARD_SOURCE_DEFINITION record aea2fd0d-377d-465e-86c0-4fdc4f688e51
airbyte-server      | 2021-07-23T04:29:43.422977100Z 2021-07-23 04:29:43 INFO i.a.c.p.DatabaseConfigPersistence(insertConfigRecord):222 - {workspace_app_root=/tmp/workspace/server/logs} - Inserting STANDARD_SOURCE_DEFINITION record af6d50ee-dddf-4126-a8ee-7faee990774f
airbyte-server      | 2021-07-23T04:29:43.424868200Z 2021-07-23 04:29:43 INFO i.a.c.p.DatabaseConfigPersistence(insertConfigRecord):222 - {workspace_app_root=/tmp/workspace/server/logs} - Inserting STANDARD_SOURCE_DEFINITION record 9e0556f4-69df-4522-a3fb-03264d36b348
airbyte-server      | 2021-07-23T04:29:43.427099100Z 2021-07-23 04:29:43 INFO i.a.c.p.DatabaseConfigPersistence(insertConfigRecord):222 - {workspace_app_root=/tmp/workspace/server/logs} - Inserting STANDARD_SOURCE_DEFINITION record 435bb9a5-7887-4809-aa58-28c27df0d7ad
airbyte-server      | 2021-07-23T04:29:43.429298100Z 2021-07-23 04:29:43 INFO i.a.c.p.DatabaseConfigPersistence(insertConfigRecord):222 - {workspace_app_root=/tmp/workspace/server/logs} - Inserting STANDARD_SOURCE_DEFINITION record 36c891d9-4bd9-43ac-bad2-10e12756272c
airbyte-server      | 2021-07-23T04:29:43.431202800Z 2021-07-23 04:29:43 INFO i.a.c.p.DatabaseConfigPersistence(insertConfigRecord):222 - {workspace_app_root=/tmp/workspace/server/logs} - Inserting STANDARD_SOURCE_DEFINITION record 68e63de2-bb83-4c7e-93fa-a8a9051e3993
airbyte-temporal    | 2021-07-23T04:29:40.551580000Z {"level":"info","ts":"2021-07-23T04:29:40.551Z","msg":"Received a ring changed event","service":"matching","component":"service-resolver","service":"worker","logging-call-at":"rpServiceResolver.go:219"}
airbyte-temporal    | 2021-07-23T04:29:40.551780200Z {"level":"info","ts":"2021-07-23T04:29:40.551Z","msg":"Current reachable members","service":"matching","component":"service-resolver","service":"worker","addresses":"[172.19.0.6:7239]","logging-call-at":"rpServiceResolver.go:266"}
airbyte-temporal    | 2021-07-23T04:29:40.614839600Z {"level":"info","ts":"2021-07-23T04:29:40.614Z","msg":"Received a ring changed event","service":"matching","component":"service-resolver","service":"matching","logging-call-at":"rpServiceResolver.go:219"}
airbyte-temporal    | 2021-07-23T04:29:40.614910000Z {"level":"info","ts":"2021-07-23T04:29:40.614Z","msg":"Received a ring changed event","service":"matching","component":"service-resolver","service":"history","logging-call-at":"rpServiceResolver.go:219"}
airbyte-temporal    | 2021-07-23T04:29:40.614987600Z {"level":"info","ts":"2021-07-23T04:29:40.614Z","msg":"Received a ring changed event","service":"matching","component":"service-resolver","service":"worker","logging-call-at":"rpServiceResolver.go:219"}
airbyte-temporal    | 2021-07-23T04:29:40.615007900Z {"level":"info","ts":"2021-07-23T04:29:40.614Z","msg":"Received a ring changed event","service":"matching","component":"service-resolver","service":"frontend","logging-call-at":"rpServiceResolver.go:219"}
airbyte-temporal    | 2021-07-23T04:29:40.670755500Z {"level":"info","ts":"2021-07-23T04:29:40.670Z","msg":"Get dynamic config","name":"frontend.enableClientVersionCheck","value":"true","default-value":"true","logging-call-at":"config.go:79"}
airbyte-temporal    | 2021-07-23T04:29:40.674441900Z {"level":"info","ts":"2021-07-23T04:29:40.674Z","msg":"Get dynamic config","name":"limit.maxIDLength","value":"1000","default-value":"1000","logging-call-at":"config.go:79"}
airbyte-temporal    | 2021-07-23T04:29:40.675743200Z {"level":"info","ts":"2021-07-23T04:29:40.675Z","msg":"none","service":"matching","component":"matching-engine","lifecycle":"Starting","wf-task-queue-name":"094e2524b586:131bc566-2583-4ec0-8b77-ab00f2968de3","wf-task-queue-type":"Workflow","logging-call-at":"matchingEngine.go:188"}
airbyte-temporal    | 2021-07-23T04:29:40.675925700Z {"level":"info","ts":"2021-07-23T04:29:40.675Z","msg":"Get dynamic config","service":"frontend","name":"matching.numTaskqueueReadPartitions","value":"4","default-value":"4","logging-call-at":"config.go:79"}
airbyte-temporal    | 2021-07-23T04:29:40.676371700Z {"level":"info","ts":"2021-07-23T04:29:40.676Z","msg":"Get dynamic config","name":"matching.outstandingTaskAppendsThreshold","value":"250","default-value":"250","logging-call-at":"config.go:79"}
airbyte-temporal    | 2021-07-23T04:29:40.676404800Z {"level":"info","ts":"2021-07-23T04:29:40.676Z","msg":"Get dynamic config","name":"matching.getTasksBatchSize","value":"1000","default-value":"1000","logging-call-at":"config.go:79"}
airbyte-temporal    | 2021-07-23T04:29:40.676777100Z {"level":"info","ts":"2021-07-23T04:29:40.676Z","msg":"none","service":"matching","component":"matching-engine","lifecycle":"Starting","wf-task-queue-name":"/_sys/temporal-sys-tq-scanner-taskqueue-0/3","wf-task-queue-type":"Workflow","logging-call-at":"matchingEngine.go:188"}
airbyte-temporal    | 2021-07-23T04:29:40.676876800Z {"level":"info","ts":"2021-07-23T04:29:40.676Z","msg":"Get dynamic config","name":"matching.forwarderMaxOutstandingTasks","value":"1","default-value":"1","logging-call-at":"config.go:79"}
airbyte-temporal    | 2021-07-23T04:29:40.676897500Z {"level":"info","ts":"2021-07-23T04:29:40.676Z","msg":"Get dynamic config","name":"matching.forwarderMaxOutstandingPolls","value":"1","default-value":"1","logging-call-at":"config.go:79"}
airbyte-temporal    | 2021-07-23T04:29:40.676933800Z {"level":"info","ts":"2021-07-23T04:29:40.676Z","msg":"Get dynamic config","name":"matching.forwarderMaxRatePerSecond","value":"10","default-value":"10","logging-call-at":"config.go:79"}
```

Reply:
@cgardens this matches what I remember